docs/basics.rst

```diff
@@ -1,39 +1,36 @@
 
-Each of these components can be deployed, run and used independently. The most convenient way of using the system is to drive it from EAGLE. EAGLE is a web application, while the *Translator* service as well as the execution *Engine* services expose RESTful interfaces, which can be used programatically or using command line tools like *curl*. In addition there is a web monitoring tool exposed by the execution engine available as well. 
+* EAGLE: The editor provides three sets of interactive tutorials in-browser. It is *highly* recommended you follow those tutorials. Go to https://eagle.icrar.org and navigate to the Help > Tutorials section on the navigation bar to review them. 
+* dlg_paletteGen: This allows you to create and load Python modules as Palettes into EAGLE, and begin creating workflows directly from Python classes, methods, and functions. 
 
-The *Editor* and the *Translator* are fairly lightweight and don't require a lot of resources. In particular the editor can be deployed locally on a user's laptop, but there is also a version running under https://eagle.icrar.org. 
 
-The |daliuge| execution engine *can* be run on a laptop as well, but, other than for testing, there is no real good use case to do this. More realistic deployment platforms are large High Performance Computing (HPC) clusters or dynamically scalable environments such as the AWS Elastic Computing Service (ECS) or a Kubernetes cluster. All three components of |daliuge| can be installed and run natively or in docker containers. 
+"Hello DALiuGE"
+---------------
+Having followed the tutorials above, as with all programming related tutorials, we encourage you to try out a simple "Hello, World" example using a BashShellAppDROP and FileDROP. 
 
-EAGLE
-#####
+To do this, you will need to: 
 
-EAGLE is a web-application allowing users to develop complex scientific workflows using a visual paradigm similar to Kepler and Taverna or Simulink and Quartz composer. It also allows groups of scientists to work together on such workflows, version them and prepare them for execution. A workflow in |daliuge| terminology is represented as a graph. In fact the |daliuge| system is dealing with a whole set of different graphs representing the same workflow, depending on the lifecycle state of that workflow. EAGLE just deals with the so-called *Logical Graph* state of a given workflow. EAGLE also offers an interface to the *Translator* and, through the *Translator* also to the *Engine*. For detailed information about EAGLE please refer to the EAGLE basic documentation under https://github.com/ICRAR/EAGLE as well as the `detailed usage documentation in readthedocs <https://eagle-dlg.readthedocs.io>`__.
+#. Create a new graph 
+#. Create a BashShellAppDROP component on the EAGLE Graph Canvas
+#. Click on the BashShellAppDrop and open the **Fields Table**
+#. In the ``command`` row, add the following to the **value** column: ``echo "Hello, World" > {output}``
```

Comment:
Given the title of this section this should rather be:
**value** column: ``echo "Hello, DALiuGE" > {output}``